Userland/Applications/PixelPaint/PaletteWidget.cpp

```diff
@@ -1,13 +1,17 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
  *
```

Comment:
For copyright, you want to add your own at the end of the list (instead of changing the copyright of other contributors) :
```suggestion
 * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
 * Copyright (c) 2021, Your Name <your@email>
```